src/Hl7.Fhir.Base/ElementModel/TypedElementOnSourceNode.cs

```diff
@@ -338,7 +342,7 @@ private string typeFromLogicalModelCanonical(ITypeSerializationInfo info)
             return pos > -1 ? type.Substring(pos + 1) : type;
         }
 
-        private bool tryGetBySuffixedName(Dictionary<string, IElementDefinitionSummary> dis, string name, out IElementDefinitionSummary info)
+        private bool tryGetBySuffixedName(Dictionary<string, IElementDefinitionSummary> dis, string name, out IElementDefinitionSummary? info)
```

Comment:
Shouldn't we have a NotNullWhen annotation here?

Reply:
<img width="1493" alt="Screenshot 2024-03-26 at 20 37 09" src="https://github.com/FirelyTeam/firely-net-sdk/assets/99174761/ede1ea94-000d-4a80-ba3a-64e084f9e384">

Reply:
I do not understand why it picks the newtonsoft here

Reply:
ohhh, I know why! we have not merged the annotations branch yet :)